packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -121,13 +121,22 @@ regen=
 # "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused8" \
 # "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused9" \
 # "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused10" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused0" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused1" \ -- This one is executed below.
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused2" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused3" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused4" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused5" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused6" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused7" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused8" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused9" \
+# "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused10" \
 
 test_list=()
 if [ "$SCRIPT_TESTS" == true ]; then
   test_list=( "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused0" \
-              "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused1(" \
               "OptimismPortalKontrol.prove_finalizeWithdrawalTransaction_paused" \
-              "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused0" \
               "OptimismPortal2Kontrol.prove_proveWithdrawalTransaction_paused1(" \
```

Comment:
is the trailing `(` correct?

Reply:
Yea this is to distinguish 1 from 10, see here: https://github.com/ethereum-optimism/optimism/pull/10292#discussion_r1578316473